DXMainClient/Domain/Multiplayer/CnCNet/TunnelHandler.cs

```diff
@@ -193,56 +211,29 @@ private List<CnCNetTunnel> RefreshTunnels()
                 }
                 catch (Exception ex)
                 {
-                    Logger.Log("Refreshing tunnel cache file failed! Returned error: " + ex.Message);
+                    Logger.Log($"Refreshing tunnel cache file failed! Returned error: {ex.Message}");
```

Comment:
Interpolation is not really clearer than concatenating when you're only concatenating a single object to the end of a string.